docs/configuration-process.asciidoc

```diff
@@ -1,31 +1,6 @@
 [[configuration-process]]
 == General configuration options
```

Comment:
Curious as to why you decided to delete the config example. Broken yaml is a common problem (at least for Beats users...not sure about APM). Showing a config example helps users understand the structure of the file. At the very least, I'd provide examples where you describe the option.

Reply:
it didn't seem too informative, as is basically a snippet copied from `apm-server.yml`
i was actually thorn on whether to remove it or not, so if you think is better to have it ill add it back, but probably smaller and showing how yml configs can look like (nested attrs, dotted attrs, etc.)

Reply:
yah, a truncated example would be fine...just something that shows structure so users know  where options are defined (whether they need to be namespaced, at the root, etc).